src/main/java/com/solvd/shop24/gui/pages/components/shop24/MenuItem.java

```diff
@@ -0,0 +1,88 @@
+package com.solvd.shop24.gui.pages.components.shop24;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+import com.solvd.shop24.gui.pages.components.shop24.constant.LocatorsConstants;
+import com.solvd.shop24.gui.pages.shop24.news.NewsPage;
+import com.solvd.shop24.gui.pages.shop24.profile.AuthorizationPage;
+import com.solvd.shop24.gui.pages.shop24.profile.ProfilePage;
+import com.solvd.shop24.gui.pages.shop24.purchase.BasketPage;
+import com.solvd.shop24.gui.pages.shop24.purchase.SearchPage;
+import com.solvd.shop24.gui.pages.shop24.purchase.SectionsCatalogPage;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class MenuItem extends AbstractUIObject {
+
+    @FindBy(xpath = "//span[text()='Вход']")
+    private ExtendedWebElement authLink;
+
+    @FindBy(className = "catalog-btn__txt")
+    private ExtendedWebElement catalogMenu;
+
+    @FindBy(xpath = ".//div[@class='top-search__input-block']/input")
+    private ExtendedWebElement searchInputField;
+
+    @FindBy(xpath = "//ul[@class='cat_nav inner-cat-nav']/li/a")
```

Comment:
try not to use conditions like `@class='cat_nav inner-cat-nav'` in xpathes.
if new class is added to web element your locator will be broken.
instead you can use `contains(@class,'cat_nav')` or better to use css locators: ul.cat_nav > li > a
reference for css locators: https://www.w3schools.com/cssref/css_selectors.asp